java/src/jmri/jmrix/loconet/locomon/Llnmon.java

```diff
@@ -222,7 +222,7 @@ private static String convertToMixed(int addressLow, int addressHigh) {
         // if we have a 2 digit decoder address, proceed accordingly
         switch (addressHigh) {
             case 0x7d:
-                log.warn("addressLow / 10 = {}", Integer.toString(addressLow / 10));
+                log.debug("addressLow / 10 = {}", Integer.toString(addressLow / 10));
```

Comment:
Please remove the ```Integer.toString(...)``` around the division. Creating a String is considered an expensive operation, and one that we want to avoid if not actually logging (i.e. if debug level logging is not enabled for this class).